Corzbank.Web/ClientApp/src/app/app.module.ts

```diff
@@ -47,10 +53,18 @@ import { ConfirmResettingComponent } from './header/authorization/forgot-passwor
     BrowserAnimationsModule,
     MatDialogModule,
     CodeInputModule,
-    MatSnackBarModule
+    MatSnackBarModule,
+    JwtModule.forRoot({
+      config:{
+        tokenGetter:tokenGetter,
+        allowedDomains:["localhost:44361"],
```

Comment:
Think about a better approach to pass the domain. Do not hardcode.

Reply:
resolved